phalcon/forms/form.zep

```diff
@@ -370,10 +370,23 @@ class Form extends Injectable implements \Countable, \Iterator
 	 */
 	public function getMessages(boolean byItemName = false) -> <Group>
 	{
-		var messages;
+		var messages, messagesByItem, elementMessage;
 
 		let messages = this->_messages;
 		if typeof messages == "object" && messages instanceof Group {
+		    /**
+		     * This part of code is for backward compatibility, it should be removed in next major version
+		     */
+		    if byItemName {
+                let messagesByItem = [];
+                for elementMessage in messages {
+                    if !isset messagesByItem[elementMessages->getField()] {
```

Comment:
it seems to me `if fetch` may be faster

Reply:
anyway here is typo `elementMessage` not `elementMessages`

Reply:
I couldn't find the way to use ```if fetch```, refer to this link https://docs.zephir-lang.com/en/latest/operators.html#fetch I thought that ```isset``` is faster than ```fetch```, please let me know if I'm wrong.